### PR TITLE
F39 webui pre blivet dialogs

### PR DESCRIPTION
**Pull request by rvykydal**
Port of https://github.com/rhinstaller/anaconda/pull/5113 to fedora-39 branch.

### CODE REVIEW

File: ui/webui/src/components/storage/InstallationMethod.jsx [hunk 1]
```diff
@@ -410,25 +431,137 @@ const InstallationDestination = ({ deviceData, diskSelection, dispatch, idPrefix
...
+                      {_("Cancel")}
+                  </Button>
+              </>
+          }>
+            <TextContent>
+                <Text component={TextVariants.p}>
+                    {_("Blivet is and advanced storage editor that lets you resize, delete, and create partitions. It can set up LVM and much more.")}
+                </Text>
+                <Text component={TextVariants.p}>
+                    {_("Changes made in Blivet will directly affect your storage.")}
```
Comment (M4rtinK) [inline ui/webui/src/components/storage/InstallationMethod.jsx]: Here as well.

**Review by M4rtinK — APPROVED** [review]
Looks good to me, thanks! :)

**Comment by AdamWill** [comment]
Can you fix the typo I mentioned in https://github.com/rhinstaller/anaconda/pull/5113#pullrequestreview-1613658662 before merging this?

**Comment by rvykydal** [comment]
> Can you fix the typo I mentioned in [#5113 (review)](https://github.com/rhinstaller/anaconda/pull/5113#pullrequestreview-1613658662) before merging this?

Done, thank you for the catch.

**Pull request merged**